VideoApi/VideoApi.Contract/Enums/UserRole.cs

```diff
@@ -9,6 +9,8 @@ public enum UserRole
         Judge = 4,
         Individual = 5,
         Representative = 6,
-        JudicialOfficeHolder = 7
+        JudicialOfficeHolder = 7,
+        QuickLinkParticipant = 8,
+        QuickLinkObserver = 9
```

Comment:
Non-quick link participants and observers are both under individual, as well as witness which. I'm not against having 2 separate ones as I know they have different functionality but is there a particular reason that is distinguished here and not elsewhere like other roles.

Reply:
That matches with the DDD ubiquitous language terms for the quick link functionality.